src/main/scala/com/itechart/ny_accidents/Application.scala

```diff
@@ -46,8 +48,12 @@ object Application extends App {
     new DistrictReportGenerator(),
     new PopulationToNumberOfAccidentsReportGenerator(populationService),
     new AccidentCountDuringPhenomenonPerHourReportGenerator(),
-    new FrequencyReportGenerator()
+    new FrequencyReportGenerator(),
+    new DetailedDistrictReportGenerator()
+
   )
 
-  reportSeq.foreach(report => NYDataDatabase.insertDataFrame(report.tableName, report.apply(mergeData, reports, creationDate)))
+  reportSeq.foreach(report => FileWriterUtils.writeToCsv(report.apply(mergedData, reports), s"reports/${report.tableName}"))
```

Comment:
Could you clarify what was the reason of replacing NYDataDatabase with FileWriterUtils?

Reply:
at this step I need csv-files, not database